sqlmesh/core/linter/definition.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+
+from sqlmesh.core.config.linter import LinterConfig
+
+from sqlmesh.core.model import Model
+
+from sqlmesh.utils.errors import raise_config_error
+from sqlmesh.core.console import get_console
+from sqlmesh.core.linter.rule import RuleSet
+
+from sqlmesh.core.config.linter import RuleListType
+
+
+def gather_rules(all_rules: RuleSet, rule_names: RuleListType) -> RuleSet:
```

Comment:
[Nit] I'd consider renaming this as `select_rules`, it's also consistent with the existing `select_models`. The verb "gather" feels less frequently used.